src/content/en/fundamentals/security/_toc.yaml

```
@@ -8,8 +8,9 @@ toc:
       path: /web/fundamentals/security/csp/
     - title: Encrypting Data In Transit
       section:
-      - title: Why HTTPS
+      - title: Why HTTPS matters
         path: /web/fundamentals/security/encrypt-in-transit/why-https
```

```suggestion
        path: https://web.dev/why-https-matters
```

Set the path to the actual new path so it goes directly there and doesn't have to go through the redirect.